lib/vanagon/engine/pooler.rb

```diff
@@ -7,7 +7,7 @@ class Pooler < Base
       attr_reader :token
 
       # The vmpooler_template is required to use the pooler engine
-      def initialize(platform, target = nil)
+      def initialize(platform, target = nil, opts = {})
```

Comment:
Works pretty much the same in Ruby. Here's an example I stole from someone else:

```ruby
def dual_argument_capturing_method(*args, **keyword_args)
  {args: args, keyword_args: keyword_args}
end

dual_argument_capturing_method(1, 2, 3, key: "value") # => {:args=>[1, 2, 3], :keyword_args=>{:key=>"value"}}
```

Still basically an options hash but a little bit less 1.9-era boilerplate. Ideally we'd just constrain our initialization so we don't have to provide a million arguments, but I'm OK shaving one yak at a time.